assets/sass/cds/_sections.scss

```diff
@@ -1033,19 +1033,19 @@ span.livefr {
 
   h2 {
     margin-top: 0;
-    font-size: 24px;
+    font-size: 20px;
```

Comment:
this is not a thing for this PR, but I would love us to move away from px sizes for fonts, and over to a rem based system as px prevents users from using their own custom style sheets to scale font if needed (like for low vision folx)